student/src/main/java/tds/student/configuration/WebConfiguration.java

```diff
@@ -0,0 +1,13 @@
+package tds.student.configuration;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.client.RestTemplate;
+
+@Configuration
+public class WebConfiguration {
+    @Bean(name = "integrationRestTemplate")
+    public RestTemplate getRestTemplate() {
+        return new RestTemplate();
```

Comment:
I did this because for now we don't need to use the AIR custom configured rest templates.  Depending on how we do our setup we may be able to continue to use the default.